tests/sentry/runner/commands/presenters/test_slackpresenter.py

```diff
@@ -74,8 +74,8 @@ def test_is_slack_enabled():
         ],
     }
 
-    assert responses.calls[1].response.status_code == 200
-    assert expected_json_data == json.loads(responses.calls[1].request.body)
+    assert responses.calls[0].response.status_code == 200
+    assert expected_json_data == json.loads(responses.calls[0].request.body)
```

Comment:
Please add another test to test the scenario where no call is made to the webhook when no update is performed. That is exactly the condition you are fixing, thus it is a good practice to add a test to ensure it does not break again.